### PR TITLE
Jeff bezanson master

### PR DESCRIPTION
**Pull request by mdcfrancis**
Added 0.4 compat to https://github.com/JeffBezanson merge

### CODE REVIEW

File: src/NamedTuples.jl [hunk 1]
```diff
@@ -194,15 +184,22 @@ macro NT( expr... )
     return make_tuple( collect( expr ))
 end
 
+# Helper function for 0.4 compat
+if VERSION < v"0.5.0" 
+    getfieldname( t, i ) = fieldnames(t)[i]
+else
+    getfieldname( t, i ) = fieldname( r, i )
```
Comment (JeffBezanson) [inline src/NamedTuples.jl]: Typo.

**Comment by JeffBezanson** [comment]
:+1:

**Comment by mdcfrancis** [comment]
Working on some internal CI issues, once those have cleared (and our tests are complete) I will merge

**Comment by davidanthoff** [comment]
@mdcfrancis Excellent, thanks! Can you also tag a new version?

**Comment by mdcfrancis** [comment]
I submitted the pull for 1.0.0

**Pull request merged**